### PR TITLE
Clarified top-up success msg 

### PR DESCRIPTION
**Pull request by arbadacarbaYK**
Common misconceptioin is that the top up is related to the funds on the funding source. 

Success msg  extended with info that correlated funds on funding source are needed and the amount is virtual until fitting.

### CODE REVIEW

**Comment by dni** [comment]
i think the explanation should be in the frontend not the API. i could also be translated there.

**Comment by arbadacarbaYK** [comment]
> i think the explanation should be in the frontend not the API. i could also be translated there.

Isnt this part the success-message that is shown in the frontend ? Thats what i wanted to edit 😃

File: lnbits/core/views/admin_api.py [hunk 1]
```diff
@@ -125,7 +125,10 @@ async def api_topup_balance(data: CreateTopup) -> dict[str, str]:
 
     await update_wallet_balance(wallet_id=data.id, amount=int(data.amount))
 
-    return {"status": "Success"}
+    return {
+        "status": "Success creating virtual funds."
+        "Payments depend on actual funds on funding source."
```
Comment (dni) [inline lnbits/core/views/admin_api.py:130]:
```suggestion
        "Payments are dependent on the actual funds available in the funding source."
```

**Comment by motorina0** [comment]
Fixed:
image

Note: i18n does not work on `admin.js` (yet).

File: lnbits/static/js/base.js [hunk 2]
```diff
@@ -144,12 +144,15 @@ window.LNbits = {
           id: wallet_id
         })
         .then(_ => {
+          credit = parseInt(credit)
           Quasar.Notify.create({
```
Comment (dni) [inline lnbits/static/js/base.js:148]: this notify should not be in base.js, this function just should return the promise

File: lnbits/static/js/admin.js [hunk 3]
```diff
@@ -199,8 +199,9 @@ new Vue({
         .then(response => {
```
Comment (dni) [inline lnbits/static/js/admin.js:199]: just like here the notification should put off here and also translated here

File: lnbits/core/views/admin_api.py (hunk 1 above)
Comment (arbadacarbaYK) [reply lnbits/core/views/admin_api.py]: > 

was extra making it short cause long msg are hardly readable but thats also good.

Comment (dni) [reply lnbits/core/views/admin_api.py]: ok

**Review by dni — APPROVED** [review]
thanks! that small wording change actually improved the code in the end :)

File: lnbits/static/js/components.js [hunk 4]
```diff
@@ -691,9 +691,25 @@ Vue.component('lnbits-update-balance', {
...
-        this.callback({value: res, wallet_id: this.wallet_id})
-      })
+      LNbits.api
+        .updateBalance(credit, this.wallet_id)
+        .then(res => {
+          this.callback({value: res, wallet_id: this.wallet_id})
+        })
+        .then(_ => {
+          credit = parseInt(credit)
+          Quasar.Notify.create({
```
Comment (dni) [inline lnbits/static/js/components.js:701]: this is bit notpicky, but couldnt it be `this.$q.notify` also like we have in other places in the code?

File: lnbits/static/i18n/en.js [hunk 5]
```diff
@@ -33,6 +33,8 @@ window.localisation.en = {
   reset_defaults_tooltip: 'Delete all settings and reset to defaults.',
   download_backup: 'Download database backup',
   name_your_wallet: 'Name your %{name} wallet',
+  wallet_topup_ok:
+    'Success creating virtual funds (%{amount} sats). Payments depend on actual funds on funding source.',
```
Comment (arbadacarbaYK) [inline lnbits/static/i18n/en.js:37]: shouldnt this say "(%{credit} sats)" in each translation ?

**Review by dni — APPROVED** [review]

File: lnbits/static/i18n/en.js (hunk 5 above)
Comment (motorina0) [reply lnbits/static/i18n/en.js:37]: no, there is an object which is passed as param:

```js
            message: this.$t('wallet_topup_ok', {
              amount: credit
            }),
```

Comment (dni) [reply lnbits/static/i18n/en.js:37]: no, amount is fine, if you look into the `this. $q.notify()`

**Review by motorina0 — APPROVED** [review]

**Pull request merged**